docs/src/plugins/remark-snackplayer.js

```diff
@@ -0,0 +1,120 @@
+const walkTree = require('./walkTree');
+
+const parseParams = (paramString = '') => {
```

Comment:
Might be helpful add some comments on what the output of this function looks like for future understanding